DEVELOPING.md

```diff
@@ -207,3 +213,35 @@ Please refer to the MAINTAINERS file for maintenance guidelines and contact info
 ## Contact and Support
 
 If you need help or have any questions, please open an issue on GitHub or contact us at support@neuralmagic.com.
+
+This will run automatic code styling using `black` and `isort` and test that the
+repository's code matches its standards.
+
+## Project configuration
```

Comment:
Can we move this section above contact and support and just after the formatting, linting, and type checking section?